en/guide/development-tools.md

```diff
@@ -7,14 +7,15 @@ description: Nuxt.js helps you to make your web development enjoyable.
 
 ## End-to-End Testing
 
-[Ava](https://github.com/avajs/ava) is a powerful JavaScript testing framework, mixed with [jsdom](https://github.com/tmpvar/jsdom), we can use them to do end-to-end testing easily.
+[AVA](https://github.com/avajs/ava) is a powerful JavaScript testing framework, mixed with [jsdom](https://github.com/tmpvar/jsdom), we can use them to do end-to-end testing easily.
```

Comment:
AVA is in capitalize as mentioned in official documentation 
— Source: https://github.com/avajs/ava